### ISSUE TITLE
The question about the loss function.

### ISSUE THREAD

**Issue by dddzg** [gyhui14/spottune#2]
Thanks for your open source. There is a question about the loss function.
In the paper, you discussed the loss function of k blocks (section 3.3). However, in the code, the loss function is the original cross entropy.  
Does it indicate that the additional losses are ineffective？

**Comment by gyhui14** [comment]
Hi dddzg, thanks for the question. The implementation considers all the blocks and can achieve better results than the k-blocks version. The additional losses have different purpose and are mainly used for dropping blocks.

**Comment by dddzg** [comment]
Thanks for your response! There is a misunderstanding due to my negligence (ﾟ∀ﾟ;).

As intuition, we still prefer a more deterministic policy even for the image-specific fine-tuning policy.
Therefore, the entropy loss (sum -v_l*log(v_l)) may be effective for the image-specific policy?

**Comment by gyhui14** [comment]
Hi dddzg, I think that depends. A deterministic policy can possibly hurt exploration of the fine-tuning policy.

**Comment by dddzg** [comment]
Thank you so much！I will close this issue right now.
